opengrok-indexer/src/main/java/org/opengrok/indexer/web/PageConfig.java

```diff
@@ -766,10 +766,10 @@ public Annotation getAnnotation() {
         if (isDir()) {
             return path.length() == 0
```

Comment:
So ... are you going to revise this line to avoid referencing the field?

Reply:
um, yes. Thanks.